framework/src/main/java/org/tron/core/net/service/effective/ResilienceService.java

```diff
@@ -137,11 +138,14 @@ private void disconnectIsolated2() {
           logger.warn("Sort disconnectIsolated2 peers failed: {}", e.getMessage());
           return;
         }
-
+        int candidateSize = peers.size();
         if (peers.size() > disconnectSize) {
           peers = peers.subList(0, disconnectSize);
         }
-        peers.forEach(peer -> disconnectFromPeer(peer, ReasonCode.BAD_PROTOCOL));
+        logger.info("All peer Size:{}, plan size:{}, candidate size:{}, real size:{}", peerSize,
```

Comment:
This log will refresh the screen.

Reply:
No, only isolated node will print this log but this condition is rare. Even in this case, its period is 30 seconds.